flopy/utils/formattedfile.py

```diff
@@ -356,10 +356,9 @@ class FormattedHeadFile(FormattedLayerFile):
     >>> import flopy.utils.formattedfile as ff
     >>> hdobj = ff.FormattedHeadFile('model.fhd', precision='single')
     >>> hdobj.list_records()
-    >>> rec = hdobj.get_data(kstpkper=(1, 50))
+    >>> rec = hdobj.get_data(kstpkper=(0, 50))
```

Comment:
Although this is a made up example wouldn't it be consistent to change (1,50) to (0,49)

Reply:
updated